docs: update troubleshoot-ksqldb.md (DOCS-6717) (#6897)
confluentinc/ksql: docs/troubleshoot-ksqldb.md

confluent local services ksql-server log
```

java.lang.NoClassDefFoundError when java.io.tmpdir is not writable
------------------------------------------------------------------

> KSQL leverages RocksDB which includes a C library. As part of the startup 

Reviewer: ```suggestion
ksqlDB leverages RocksDB, which includes a C library. As part of the startup 
```

process of RocksDB, it has to extract the C library before it can be used. The 
location to extract the C library is determined by the java.io.tmpdir system 
property or `ROCKSDB_SHAREDLIB_DIR` environment variable. If this directory is 

> not writable, then the user will see an error like the following

Reviewer: ```suggestion
not writable, you'll see an error like the following:
```
